doc/classes/Callable.xml

```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="Callable" version="4.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
-		An object representing a method in a certain object that can be called.
+		Built-in type representing a object instance's method that can be called.
 	</brief_description>
 	<description>
-		[Callable] is a first class object which can be held in variables and passed to functions. It represents a given method in an [Object], and is typically used for signal callbacks.
+		[Callable] is a built-in [Variant] type that represents a given method in an [Object] instance. Like all [Variant] types, it can be stored in variables and passed to functions. It is most commonly used for signal callbacks.
```

Comment:
```suggestion
		[Callable] is a built-in [Variant] type that represents a given function, which can be a method in an [Object] instance or a standalone function not associated with any object, like a lambda function. Like all [Variant] types, it can be stored in variables and passed to functions. It is most commonly used for signal callbacks.
```